pysd/py_backend/utils.py

```diff
@@ -6,32 +6,37 @@
 
 import os
 import warnings
-import keyword
 import json
 from collections.abc import Mapping
 
 import regex as re
 import progressbar
 import numpy as np
 import xarray as xr
+import pandas as pd
 
-# used to create python safe names
+# used to create python safe names with the variable reserved_words
+from keyword import kwlist
+from builtins import __dir__ as bidir
 from .decorators import __dir__ as ddir
 from .external import __dir__ as edir
 from .functions import __dir__ as fdir
+from .statefuls import __dir__ as sdir
 
 
-def xrmerge(das, accept_new=True):
+reserved_words = set(dir() + fdir() + edir() + ddir() + sdir() + bidir())
```

Comment:
This is a clever way to make sure we don't overwrite pysd-specific function names. I like it.